### DISCUSSION TITLE
Is it possible to open 2 URL/window tabs  at the same in Safe Exam Browser during the exam

### DISCUSSION THREAD

**Discussion by mufasi4** [SafeExamBrowser/seb-win-refactoring#601] [Q&A]
Is it possible to open 2 URLs at the same in Safe Exam Browser during the exam?

If Possible 

Please suggest the steps involved in it.

**Comment by dbuechel** [comment]
Yes, you certainly can open various browser windows within SEB, be it via a link which is to be opened in a new tab / window or via JavaScript. What is not possible at the moment is opening two URLs at the same time, i.e. setting two start URLs in the configuration or alike.
